static/app/utils/discover/fields.tsx

```diff
@@ -83,27 +83,27 @@ export type Alignments = 'left' | 'right';
 
 const CONDITIONS_ARGUMENTS: SelectValue<string>[] = [
   {
-    label: 'equal =',
+    label: 'is equal to',
     value: 'equals',
   },
   {
-    label: 'not equal !=',
+    label: 'is not equal to',
     value: 'notEquals',
   },
   {
-    label: 'less <',
+    label: 'is less than',
     value: 'less',
   },
   {
-    label: 'greater >',
+    label: 'is greater than',
     value: 'greater',
   },
   {
-    label: 'less or equals <=',
+    label: 'is less than or equal to',
     value: 'lessOrEquals',
   },
   {
-    label: 'greater or equals >=',
+    label: 'is greater than or equal to',
```

Comment:
So I went with these shorter descriptions cause of how it looks in the 3-column situation:
<img width="726" alt="image" src="https://user-images.githubusercontent.com/4205004/125151015-274dc080-e112-11eb-9e8f-0392fc94811a.png">